transformations/chinese_simplified_traditional_perturbation/transformation.py

```diff
@@ -0,0 +1,77 @@
+import itertools
+import random
+import opencc
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+
+def chinese_simplified_traditional_perturbation(text,
+                          prob,
+                          seed,
+                          max_outputs,
+                          config
+                          ):
+    random.seed(seed)
+
+    perturbed_texts = []
+
+    if(config == 's2t'):
+        converter = opencc.OpenCC('s2t.json')
+    elif (config == 't2s'):
+        converter = opencc.OpenCC('t2s.json')
+    else:
+        print("Configuration not specified")
+    for _ in itertools.repeat(None, max_outputs):
+        butter_text = ""
+        for chinese_character in text:
+            if random.random() <= prob:
+
+                new_chinese_character = converter.convert(chinese_character)
+            else:
+                new_chinese_character = chinese_character
+
+            butter_text += new_chinese_character
+        perturbed_texts.append(butter_text)
+    return perturbed_texts
+
+
+
+
+"""
+Chinese Words and Characters Butter Fingers Perturbation
+"""
+
+class ChineseSimplifiedTraditionalPerturbation(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION
+    ]
+    languages = ["zh"]
+
+    def __init__(self, seed=0, max_outputs=1, prob=1, config = 's2t'):
+        super().__init__(seed, max_outputs=max_outputs)
+        self.prob = prob
+        self.seed = seed
+        self.max_outputs = max_outputs
+        self.config = config
+
+    def generate(self, sentence: str):
+        perturbed_texts = chinese_simplified_traditional_perturbation(
```

Comment:
You can simplify by just doing the return here.

As the function does nothing, maybe put the content of `chinese_simplified_traditional_perturbation` in this function directly.

Reply:
I'll keep it this way for now since if it is extended in the future then it might be useful to keep it separate. What do you think?

Reply:
It depends, 
Try to keep it simple for the moment and suggest amelioration in the Readme.
I believe that it is not a good habit to think to much of extension in the main branch. 
Those might never be done or coded and your code is complexified for no reason.
But that is my opinion :)

Reply:
I can understand your point! I've made the changes since it's a quick change. :)